CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Command-line arguments are now only available in non-shipping builds, if you wish to use command-line arguments for shipping builds the target rule `bEnableSpatialCmdlineInShipping` will let you do so.
 - Dynamic Worker Flags are once again supported with the Standard Runtime Variant. 
 - Simulated Player deployments started with the DeploymentLauncher now startup faster thanks to Dynamic Worker Flags. DeploymentLauncher `createsim` usage has been updated to include the new boolean argument `<auto-connect>` which will automatically connect your sim players to your deployment when it is ready.  
+- Schema bundles are now generated by default (in addition to the schema descriptor). Specifying a Schema bundle location via `-AdditionalschemaCompilerArguments` will result in the bundle being written to both the default location (`build/assembly/schema/schema.sb`) _and_ the location specified by the additional arguments.
```

Comment:
```suggestion
- Schema bundles are now generated by default (in addition to the schema descriptor). Specifying a Schema bundle location via `-AdditionalSchemaCompilerArguments` will result in the bundle being written to both the default location (`build/assembly/schema/schema.sb`) _and_ the location specified by the additional arguments.
```
?

Reply:
Also, this may be slightly confusing, as I assume you mean that specifying `-AdditionalschemaCompilerArguments="--bundle_out=somewhere"` would actually output it to that location? It's two layers of args.

Reply:
Perhaps we should mention why we are doing this?

Reply:
Didn't mean to mark it as resolved. It auto did that after I committed the suggestion. Will reword this too.
The reason for doing this is mostly internal right now. Happy to discuss what might be useful to put here though.